pkg/kubelet/pleg/generic.go

```diff
@@ -406,9 +406,9 @@ func (g *GenericPLEG) updateCache(pod *kubecontainer.Pod, pid types.UID) error {
 	// all containers again.
 	status, err := g.runtime.GetPodStatus(pod.ID, pod.Name, pod.Namespace)
 	if klog.V(6).Enabled() {
-		klog.V(6).ErrorS(err, "PLEG: Write status", "pod", klog.KRef(pod.Namespace, pod.Name), "podStatus", status)
```

Comment:
Error messages don't have a verbosity. One could argue that these should be info calls with an `err` key for the error instead of error calls?